### PR TITLE
OP-999 make jasper reports loadable from folder

### PR DESCRIPTION
**Pull request by nlazzer1**
See OP-999.

### CODE REVIEW

File: src/main/java/org/isf/stat/manager/JasperReportsManager.java [hunk 1]
```diff
@@ -977,22 +993,58 @@ private JasperReportResultDto generateJasperReport(String jasperFilename, String
...
+	}
+
+	private String compileFilename(String folderName, String jasperFileName, List<String> params, String ext) {
+		StringBuilder sbFilename = new StringBuilder();
+		sbFilename.append(folderName);
+		sbFilename.append(File.separator);
+		sbFilename.append(jasperFileName);
+		if(params != null) 
+		{
+			params.forEach(p -> {
```
Comment (dbmalkovsky) [inline src/main/java/org/isf/stat/manager/JasperReportsManager.java:1012]: I had to hide white space differences to better see the changes in the file.   It looks like there was a fair amount of reformatting done.    In addition like things like `if(` should be `if (` to match project standards.   And braces are not on their own line, so line 1010 should be `if (parms != null) {`

Comment (nlazzer1) [reply src/main/java/org/isf/stat/manager/JasperReportsManager.java:997]: Thanks David, I reformatted the code according to the OH style

File: pom.xml [hunk 2]
```diff
@@ -131,7 +131,7 @@
 								<exclude>lib/**</exclude>
 								<exclude>sql/**</exclude>
 								<exclude>repo/**</exclude>
-								<exclude>rpt/**</exclude>
+								<exclude>rpt_base/**</exclude>
```
Comment (mwithi) [inline pom.xml:134]: should we add here also the other new folders?

**Review by mwithi — APPROVED** [review]
LGTM. Thanks a lot!

**Pull request merged**